app/src/common/shared/com/igalia/wolvic/browser/PermissionDelegate.java

```diff
@@ -82,12 +82,32 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    private @SitePermission.Category int toSitePermission(final PermissionWidget.PermissionType aType) {
+        // So far we only handle Location.
+        if (aType == PermissionWidget.PermissionType.Location)
```

Comment:
I thought about that but it's unlikely that we'd end up doing anything with the other values, so in order to avoid
 ```
switch () {
case a:
  do something;
case b:
case c:
case d:
  do nothing
}
```
I decided to use an if clause.